### DISCUSSION TITLE
Set sketchybarrc to use

### DISCUSSION THREAD

**Discussion by neutonfoo** [FelixKratz/SketchyBar#298] [General]
I was wondering if there was a way set a sketchybarrc file. I have a MacBook Air with a notch and occasionally use it with and without monitors. I have created two version of my bar, one that accomadates the notch, and one that doesn't, and was wondeirng if there was a way to set the sketchybarrc file used by Sketchybar via CLI. 

I'm writing a bash script to detect if I have my monitors connection and was hoping to set the sketchybarrc and reload

Edit: Just realized I could just put add my conditional in sketchybarrc

**Comment by FelixKratz** [comment]
Are those two setups so drastically different that you need to restart sketchybar? You could make those changes without restarting sketchybar, e.g. by removing all items and manually invoking a different sketchybarrc:
```bash
sketchybar --remove '/.*/' && sh -c '$HOME/.config/sketchybar/sketchybarrc_alternative'
```

**Reply by neutonfoo** [reply]
Wow this is a much better idea than what I was doing. Thanks! (Had to `source` sketchybarrc but it works perfectly now, with the exception of some global/default options seemingly being cached.)

**Reply by FelixKratz** [reply]
The command `--remove` removes an item from sketchybar and it also takes a regex as an argument, so `sketchybar --remove '/.*/'` removes all items from sketchybar. It does not reset the bar or the defaults, but I could imagine having a command to completely reset sketchybar without having to restart it. Possibly: `sketchybar --reset`
